packages/react/src/components/Masthead/Masthead-test.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Masthead from './Masthead';
+import { shallow } from 'enzyme';
```

Comment:
Probably not important for this commit, but wondering if we should use storyshots to rely more on snapshot testing over enzyme?

Reply:
Sure we could set that up later. This is just a component stub required for the project to build.